app/src/main/res/layout/item_failed.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you rename this file `failed_iterm.xml` for consistency with the class name?

Reply:
I called this item this way to be consistent with the rest of the project. 
<img width="295" alt="Screenshot 2020-01-27 at 10 46 51" src="https://user-images.githubusercontent.com/1616097/73149103-5b861a80-40f2-11ea-9604-225cfb9f02aa.png">

And the naming of Android studio: `SettingsFragment` always has a `fragment_settings.xml` file

Reply:
👌